crates/flow-pkt/release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# This script is used to release the IANA information elements and protocol numbers as xml files
+set -exm
+
+# Get the current date
+current_date=$(date '+%Y-%m-%d')
+
+# Load the iana information elements from IANA registry as xml
+curl https://www.iana.org/assignments/ipfix/ipfix.xml > iana_ipfix_information_elements.xml
+
+# Insert the comment with the current date after the first line
+{
+  head -n 1 iana_ipfix_information_elements.xml  # Print the first line (the XML declaration)
+  echo "<!-- Date of download: $current_date -->"  # Print the comment with the current date
+  tail -n +2 iana_ipfix_information_elements.xml   # Print the rest of the file starting from line 2
+} > temp.xml && mv temp.xml iana_ipfix_information_elements.xml
+
+# Move the iana_ipfix_information_elements.xml file to registry/subregistry folder
+mv iana_ipfix_information_elements.xml crates/flow-pkt/registry
```

Comment:
I aslo get an error on this line 
```bash
+ mv iana_ipfix_information_elements.xml crates/flow-pkt/registry
mv: rename iana_ipfix_information_elements.xml to crates/flow-pkt/registry: No such file or directory
```

Reply:
You are correct—if release.sh is run from its current location, it won't work as intended. Now that it's fixed, I realize I initially placed the script there because I wasn't sure about your release process. Since you maintain multiple crates in a single repository, I assumed that they are always released together. That said, I'm still unsure whether release.sh should be in the root of the repo or if its current location is appropriate.

Additionally, the release workflow I’ve set up is more of a suggestion, as I’m not fully familiar with your current process.

Reply:
for the release process I'm doing it manually at the moment, since I didn't figure out a good way to release a cargo workspace project. I spent sometime with some of the existing tools (e.g, cargo-releae and cargo-smart-release), but never got around to learn them and make good use of them. It's also one of my lower-priority todo items.